vision/image_simulator/create_letter_samples.py

```diff
@@ -60,9 +54,10 @@ def main(jobs):
     os.makedirs(dst_folder)
 
     img_index = 0
+    print gs.LETTER_LABELS
     for img_path, data_path in zip(imgs_paths, data_paths):
 
-        print('Extracting patches from image', img_path)
+        print 'Extracting patches from image', img_path
```

Comment:
May need to rebase w/ master cause I think I fixed these in a recent python3 commit.